USAGE.md

```diff
@@ -0,0 +1,260 @@
+# Usage
+
+Below, you can find a selection of the most used functionality of the Connect Go SDK. For more detailed information about the content of the SDK, please refer to the [GoDocs](https://pkg.go.dev/github.com/1Password/connect-sdk-go).
+
+## Creating a Connect API Client
+
+A 1Password Connect client (`connect.Client`) is required to make requests to the Connect server via the 1Password Go SDK.
+The client is configured with a token and a hostname. Three constructor methods that allow for creating the 1Password Connect client are provided.
+
+- `connect.NewClient` – Accepts a hostname and a token value.
+
+  ```go
+  package main
+
+  import "github.com/1Password/connect-sdk-go/connect"
+
+  func main () {
+      client := connect.NewClient("<your_connect_host>", "<your_connect_token>")
+  }
+  ```
+
+- `connect.NewClientFromEnvironment` – Fetches the hostname and token value from the environment, and expects these to be passed as environment variables (`OP_CONNECT_HOST` and `OP_CONNECT_TOKEN`, respectively).
+
+Assuming that `OP_CONNECT_TOKEN` and `OP_CONNECT_HOST` have been set as environment variables, the `connect.NewClientFromEnvironment` can be invoked as such:
+
+```go
+package main
+
+import "github.com/1Password/connect-sdk-go/connect"
+
+func main () {
+	client, err:= connect.NewClientFromEnvironment()
+	if err != nil {
+		panic(err)
+	}
+}
+```
+
+- `connect.NewClientWithUserAgent` – Accepts a hostname, a token value, and a custom User-Agent string for identifying the client to the 1Password Connect API:
+
+```go
+package main
+
+import "github.com/1Password/connect-sdk-go/connect"
+
+func main () {
+	client := connect.NewClientWithUserAgent("<your_connect_host>", "<your_connect_token>", "Mozilla/5.0 (iPhone; CPU iPhone OS 10_3_2 like Mac OS X) AppleWebKit/603.2.4 (KHTML, like Gecko) FxiOS/8.1.1b4948 Mobile/14F89 Safari/603.2.4")
+}
+```
+
+## Working with Vaults
+
+```go
+// Get a list of all vaults
+vaults, err := client.GetVaults()
+if err != nil {
+	log.Fatal(err)
+}
+
+// Get a specific vault
+vault, err := client.GetVault("vaultID _or_ vaultTitle")
+if err != nil {
+	log.Fatal(err)
+}
+
+vaultByID, err := client.GetVaultById("valutID")
+if err != nil {
+    log.Fatal(err)
+}
+
+vaultByTitle, err := client.GetVaultByTitle("valutTitle")
+if err != nil {
+    log.Fatal(err)
+}
+```
+
+## Working with Items
+
+### CRUD operations on items
+
+```go
+vault := "vaultID _or_ vaultTitle"
+
+// Get a list of all items in a vault
+items, err := client.GetItems(vault)
+if err != nil {
+	log.Fatal(err)
+}
+
+// Create an item
+newItem := &onepassword.Item{
+	Title:    "Secret String",
+	Category: onepassword.Login,
+	Tags:     []string{"1password-connect"},
+	Fields: []*onepassword.ItemField{{
+		Value: "mysecret",
+		Type: "STRING",
```

Comment:
The values for these keys use a mix of spacing (some of them are aligned, others not). Should they be made consistent here?

Reply:
Indeed, it should be consistent. Nice catch. It seems that the formatting didn't work well here.